platform-tooling-support-tests/src/main/java/platform/tooling/support/Request.java

```diff
@@ -37,16 +40,47 @@
 	private static final Path TOOLS = Paths.get("build", "test-tools");
 	public static final Path WORKSPACE = Paths.get("build", "test-workspace");
 
-	private static final String MAVEN_VERSION = "3.6.1";
+	private static final String ANT_VERSION = "1.10.12";
+	private static final String MAVEN_VERSION = "3.8.5";
 
 	public static Builder builder() {
 		return new Builder();
 	}
 
+	public static Ant ant() {
+		// Locate pre-installed And before downloading and installing it
+		var antHome = System.getenv("ANT_HOME");
```

Comment:
Is this set on GitHub Actions runners?

Reply:
No. But it could be present on other systems, and therefore used instead of downloading it.

Reply:
Remove for the being.